### ISSUE TITLE
Request: Show filter rules that are being applied

### ISSUE THREAD

**Issue by gohamstergo** [uBlock-LLC/uBlock#171]
Perhaps where ublock says, for example: "on this page 5 or 2%" you could click on the 5 and it would popup a list showing what elements are actually being blocked. It's nothing super important, I just think it'd be cool to see what's being blocked, in addition to the number and percentage.

Thanks and keep up the good work :D

**Comment by gorhill** [comment]
You can already see this in the _Statistics_ tabs, if you enable the logging of requests -- disabled by default since most users won't care and this would consume more memory.

**Comment by Mikey1993** [comment]
Well, if are concerned about memory, maybe give the user the ability to enable this feature (set it disabled by default), or is just if the feature is implemented (and not enabled) might consume more memory?

**Comment by Baegus** [comment]
But that's how it is already. Check out the Statistics tab.

**Comment by Mikey1993** [comment]
But it's not convenient for a novice user to debug why he is having a site breakage.

**Comment by Baegus** [comment]
Well, if he knows how to read the log, he probably knows how to access the plugin's settings.

**Comment by Mikey1993** [comment]
We are talking here about having displayed for the user a list with all the blocked elements on a the page via a handy balloon/popup menu or whatever.

This can be combined with a feature of disabling a rule (ability of most blockers) in a single mouse click conveniently (add it to the whitelist list) either from the statistics tab or some other menu.

**Comment by gorhill** [comment]
> We are talking here about having displayed for the user a list with all the blocked elements on a the page via a handy balloon/popup menu or whatever

I have already closed issues which were proposing this, for reason of feature creep: I figured that if I start to show the blocked requests in the popup (which I also don't like because clutter the popup with rather technical information), then users will want ability to turn off one single filters, etc etc.

I want to keep the main UI simple, and whoever has an interest in more technical info, that's the purpose of the _Statistics_ tab. I just can't let feature creep take over, I am going to be very conservative with it (and this also allow me to not look up to an endless pile of work to do).

There is an issue already filed to be able to create an exception filter from a blocked net request in the _Statistics_ tab (or the reverse, a block filter from an allowed request).

I really do not intend at all the clutter the popup, if anything , I want it **less** cluttered -- by finding a way to remove the text explaining the "power button" from it.

Anyone wanting more technical gadgetry, I suggest to install HTTP Switchboard.
